e2e/runner/setup_evm.go

```diff
@@ -167,4 +169,42 @@ func (r *E2ERunner) SetupEVM(contractsDeployed bool, whitelistERC20 bool) {
 	// We use this config to be consistent with the old implementation
 	// https://github.com/zeta-chain/node-private/issues/41
 	require.NoError(r, config.WriteConfig(ContractsConfigFile, conf))
+
+	// chain params will need to be updated if they do not match the default params
```

Comment:
Maybe we can always send `MsgUpdateChainParams` and skip check.
This might simplify code for very small additional setup time

Reply:
Updated